tests/v2/validation/rbac/rbac_test.go

```diff
@@ -284,6 +289,12 @@ func (rb *RBTestSuite) ValidateRemoveProjectRoles() {
 	err := users.RemoveProjectMember(rb.client, rb.standardUser)
 	require.NoError(rb.T(), err)
 
+	log.Info("Validating PRTB is deleted for the user.")
+	prtbs, err := rb.client.Steve.SteveType("management.cattle.io.projectroletemplatebinding").List(nil)
+	require.NoError(rb.T(), err)
+	_, err = CheckRBACResourceExistsForUser(rb.client, prtbs, rb.standardUser.Username)
+	require.Error(rb.T(), err)
+	log.Info(err.Error())
```

Comment:
Don't have a strong opinion here but we don't have to log the error here.

Reply:
Removed